tests/testthat/test_loo_moment_matching.R

```diff
@@ -0,0 +1,303 @@
+library(loo)
+library(rstan)
+options(mc.cores = 1)
+
+
+context("moment matching")
+set.seed(123)
+S <- 40000
```

Comment:
do we need that many posterior draws?

Reply:
Maybe that's a bit excessive :) Reduced to 4000.